src/platforms/esp32/partitions.csv

```diff
@@ -8,5 +8,6 @@
 nvs,      data, nvs,       0x9000,     0x6000,
 phy_init, data, phy,       0xf000,     0x1000,
 factory,  app,  factory,  0x10000,   0x1C0000,
-boot.avm,  data, phy,     0x1D0000,    0x40000,
-main.avm, data, phy,     0x210000,   0x100000
+boot.avm,  data, phy,     0x1D0000,    0x80000,
+main.avm, data, phy,     0x250000,   0x80000
+fatpart, data, phy, 0x2D0000, 0x100000
```

Comment:
think changes here were accidentally committed?

Reply:
@petermm: honestly it is still totally WIP, I just pushed stuff for having a few CI runs ;)